packages/smart-wallet/test/devices.js

```diff
@@ -10,7 +10,6 @@ export const devices = {
           case 'centralSupply':
             return bundleCentralSupply;
           case 'mintHolder':
-            return bundleMintHolder;
```

Comment:
stray edit. I think you wanted to remove the next line (which needs removing for this PR )